src/include/timestamp.i

```diff
@@ -52,6 +52,39 @@ __wt_time_window_copy(WT_TIME_WINDOW *dest, WT_TIME_WINDOW *source)
     *dest = *source;
 }
 
+/*
+ * __wt_time_window_clear_obsolete --
+ *     Where possible modify time window values to avoid writing obsolete values to the cell later.
+ */
+static inline void
+__wt_time_window_clear_obsolete(WT_SESSION_IMPL *session, WT_TIME_WINDOW *tw, WT_RECONCILE *r)
```

Comment:
It's not right to pass a WT_RECONCILE structure in here - that structure is specific to the reconciliation code. It would be better to either pass in the time pair directly as parameters.